webapp/packages/plugin-app-logo-administration/src/PluginBootstrap.ts

```diff
@@ -7,18 +7,18 @@
  */
 import { Bootstrap, injectable } from '@cloudbeaver/core-di';
 import { AdministrationTopAppBarService, WizardTopAppBarService } from '@cloudbeaver/plugin-administration';
-import { Logo } from '@cloudbeaver/plugin-top-app-bar';
+import { Logo } from '@cloudbeaver/plugin-app-logo';
 
 @injectable()
-export class AdministrationTopAppBarBootstrap extends Bootstrap {
+export class PluginBootstrap extends Bootstrap {
```

Comment:
it's better to keep name because it's really hard to debug problems in plugin bootstrap services (cuz it's a lot of them)